Fenner_William-DFBT/js/index.js

```diff
@@ -4,10 +4,15 @@
 
 function form_builder(attach, ...form_elements){
 
+    //Create a form element
     var form = document.createElement('form');
 
+    //Loop through all form elements from an array
     for(var i=0; i<form_elements.length; i++){
 
+        //Create an element based on the array input.
+        //input will create a text based input field
+        //button will create a button based on the type specified, etc
         if(form_elements[i][0] == "input"){
```

Comment:
Consider a series of ternary operators to reduce code.`form.innerHTML += form_elements[i][0] === "label" : someElement : ""`